feature/interface/aggregate/otg_tests/aggregate_all_not_viable_test/aggregate_all_not_forwarding_viable_test.go

```diff
@@ -0,0 +1,1025 @@
+	intf := isisPath.Interface(intfName)
+	query := intf.LevelAny().AdjacencyAny().AdjacencyState().State()
+	_, ok := gnmi.WatchAll(t, dut, query, time.Minute, func(val *ygnmi.Value[oc.E_Isis_IsisInterfaceAdjState]) bool {
+		v, ok := val.Val()
+		return v == state && ok
+	}).Await(t)
+
+	return ok
+
+}
+
+// checkBidirectionalTraffic verify the bidirectional traffic on DUT ports.
+func checkBidirectionalTraffic(t *testing.T, dut *ondatra.DUTDevice, portList []*ondatra.Port) error {
+
+	for _, port := range portList {
+		txPkts := gnmi.Get(t, dut, gnmi.OC().Interface(port.Name()).Counters().OutPkts().State())
+		rxPkts := gnmi.Get(t, dut, gnmi.OC().Interface(port.Name()).Counters().InPkts().State())
+		if got := (rxPkts - rxPktsBeforeTraffic[port]) / 100; got == 0 {
+			return fmt.Errorf("No Packet received, LossPct on Port %s: got %d", port.Name(), got)
+		}
+		if got := (txPkts - txPktsBeforeTraffic[port]) / 100; got == 0 {
+			return fmt.Errorf("No Packet transmitted, LossPct on Port %s: got %d", port.Name(), got)
+		}
+	}
+	return nil
+}
+
+// confirmNonViableForwardingTraffic verify the traffic received on DUT
+// interfaces and transmitted to ATE-1
+func confirmNonViableForwardingTraffic(t *testing.T, dut *ondatra.DUTDevice, ate *ondatra.ATEDevice,
+	atePort []*ondatra.Port, dutPort []*ondatra.Port) error {
+
+	// Ensure no traffic is transmitted out of DUT ports with Forwarding Viable False
+	for _, port := range atePort {
+		rxPkts := gnmi.Get(t, ate.OTG(), gnmi.OTG().Port(port.ID()).Counters().InFrames().State())
+		if got := rxPkts / 100; got > 0 {
+			return fmt.Errorf("Packets are transmiited out of %s: got %d, want 0", port.Name(), got)
+		}
+	}
+	// Ensure that traffic is delivered to ATE-1 port1
+	for _, port := range dutPort {
+		rxPkts := gnmi.Get(t, dut, gnmi.OC().Interface(port.Name()).Counters().InPkts().State()) - rxPktsBeforeTraffic[port]
+		txPkts := gnmi.Get(t, dut, gnmi.OC().Interface(dutPortList[0].Name()).Counters().OutPkts().State()) - txPktsBeforeTraffic[port]
+		if got := rxPkts / 100; got == 0 {
+			return fmt.Errorf("No Packet received on Interface %s: got %d, want packet", port.Name(), got)
+		}
+		if got := txPkts / 100; got == 0 {
+			return fmt.Errorf("No Packet transmitted on Interface %s: got %d, want packet", port.Name(), got)
+		}
+	}
+	return nil
+}
+
+// validateLag3Traffic to ensure traffic Received/Transmitted on DUT LAG_3
+func validateLag3Traffic(t *testing.T, dut *ondatra.DUTDevice, ate *ondatra.ATEDevice, dutPortList []*ondatra.Port) bool {
+	result := false
+	for _, port := range dutPortList {
+		rxPkts := gnmi.Get(t, dut, gnmi.OC().Interface(port.Name()).Counters().InPkts().State()) - rxPktsBeforeTraffic[port]
+		txPkts := gnmi.Get(t, dut, gnmi.OC().Interface(port.Name()).Counters().OutPkts().State()) - txPktsBeforeTraffic[port]
+
+		if got := rxPkts / 100; got > 0 {
```

Comment:
same here , the check is too liberal

Reply:
Verifying same here as well for the packet not rx/tx more than 100